setup.cfg

```diff
@@ -46,7 +46,7 @@ packages = find:
 include_package_data = True
 
 install_requires =
-    equistore @ https://github.com/lab-cosmo/equistore/archive/467baf0.zip
+    equistore @ https://github.com/lab-cosmo/equistore/archive/a554ce4.zip
```

Comment:
You may want to use something from before https://github.com/lab-cosmo/equistore/pull/85 until I also update rascaline